python/foglamp/services/core/service_registry/service_registry.py

```diff
@@ -87,7 +86,9 @@ def _remove_from_scheduler_records(cls, service_name):
         if service_name in ("FogLAMP Storage", "FogLAMP Core"):
             return
 
-        future = asyncio.ensure_future(server.Server.scheduler.remove_service_from_task_processes(service_name))
+        # Require a local import in order to avoid circular import references
+        from foglamp.services.core.scheduler.scheduler import Scheduler
```

Comment:
we should not do import like this!

Reply:
why not, specially in the light of circular imports?